libcudacxx/test/libcudacxx/std/cccl/128bit_types.compile.pass.cpp

```diff
@@ -0,0 +1,27 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+#include <cuda/std/__cccl/128bit_types.h>
```

Comment:
Tests of internal machinery are conventionally located in libcudacxx/test/libcxx

We should maybe rename that to `libcudacxx/test/cccl` but yeah